rafs/src/metadata/layout/v5.rs

```diff
@@ -914,7 +914,7 @@ pub struct RafsV5Inode {
     pub i_digest: RafsDigest, // 32
     /// parent inode number
     pub i_parent: u64,
-    /// from fs stat()
+    /// rafs inode number, set_ino() from RAFS_ROOT_INODE = 1
```

Comment:
The comment is a little confusing, could you help to elaberate more?

Reply:
src_ino is rafs inode number, but not from fs stat(). 
src_ino is set by function set_ino(), and its number is from const RAFS_ROOT_INODE = 1.